Contribute/TOC.md

```diff
@@ -25,5 +25,6 @@
 ## Writing essentials 
 ### [Markdown](how-to-write-use-markdown.md)
 ### [Links](how-to-write-links.md)
+### [Docs Authoring Pack](how-to-write-docs-auth-pack.md)
```

Comment:
Switch to sentence case: "Docs authoring pack"

Reply:
@BillWagner "Docs Authoring Pack" is a proper noun- it's the name of the extension pack (actually it was originally lower case but the GPM of VS Code asked us to capitalize it) - see https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack